lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc

```diff
@@ -0,0 +1,67 @@
+= Fitting a Model: College Degrees v. Income
+++++
+<style>
+.studentAnswerMedium { min-width: 8em !important; }
+td { padding: 0 !important; }
+body.workbookpage .fitbruby { padding-top: 1.1rem; }
+body:not(.LessonPlan, .narrativepage) p { line-height: 1.2; }
+.center { padding: 0; }
+</style>
+++++
+
+For this page, you'll need to the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+== Build a Model from Samples
+
+**Make a scatter plot showing the relationship between `college-or-higher` and `median-household-income`.**
+
+@n Record the `college-or-higher` and `median-household-income` values for Alabama (AL) and Alaska (AK), as (x,y) pairs below:
+
+@center{
+ (@fitbruby{10em}{@ifsoln{22.6}}{AK college-or-higher}, @fitbruby{10em}{@ifsoln{43253}}{AK median-household-income}) @hspace{10em} (@fitbruby{10em}{@ifsoln{27.5}}{AL college-or-higher}, @fitbruby{12em}{@ifsoln{70760}}{AL median-household-income})
+}
+
+@n Using the two points above, compute the slope and y-intercept to define a linear model (in Function Notation and Pyret) by filling in the blanks below. Show your work!
+
```

Comment:
I think we need the same kind of scaffolding here. I wouldn't expect students to remember how to find the slope, nor how to find the y-intercept from 2 points.

Reply:
I really feel like this level of scaffolding is overkill. But I'm thinking there's another argument for including it that you just reminded me of....